waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/metrics/MonitoredAspect.java

```diff
@@ -70,24 +71,20 @@ public Object monitor(ProceedingJoinPoint pjp, Monitored monitored) throws Throw
   }
 
   @VisibleForTesting
-  void setCounterService(CounterService counterService) {
-    this.counterService = counterService;
-  }
-
-  @VisibleForTesting
-  void setGaugeService(GaugeService gaugeService) {
-    this.gaugeService = gaugeService;
+  void setMeterRegistry(MeterRegistry meterRegistry) {
+    this.meterRegistry = meterRegistry;
   }
 
   private void increment(String metricName) {
-    if (counterService != null) {
-      counterService.increment(metricName);
+    if (meterRegistry != null) {
+      meterRegistry.counter(metricName).increment();
+
```

Comment:
Is this the unnecessary blank line added to catch me out?

Reply:
😄 I have removed it !